.github/workflows/ci-docker-builds.yml

```diff
@@ -0,0 +1,133 @@
+# Builds Pimcore Docker images. Useful for verification in continuous integration scenarios.
+# on:
+#   pull_request:
+#     types: [opened, reopened, synchronize]
+# jobs:
+#   build-images:
+#     uses: TorqIT/pimcore-github-actions-workflows/.github/workflows/ci-docker-builds@v6
+#
+# Requires the following permissions to be set in the callig job:
+# permissions:
+#   contents: read
+#   actions: read
+#
```

Comment:
Do we delete these comments?

Reply:
These are documentation so the person calling the workflow knows how to use it

Reply:
Ok sounds good